firefox-ios/Providers/Profile.swift

```diff
@@ -602,11 +602,23 @@ open class BrowserProfile: Profile {
 
         // send all unsent close tab commands
         self.tabs.getUnsentCommandUrlsByDeviceId(deviceId: deviceId) { urls in
-            constellation.sendEventToDevice(targetDeviceId: deviceId, e: .closeTabs(urls: urls))
+            constellation.sendEventToDevice(targetDeviceId: deviceId,
+                                            e: .closeTabs(urls: urls)) { result in
```

Comment:
Super-minor nitpick but it might be nice at some point to update this with a more descriptive arg label.